### ISSUE TITLE
Output file not valid JSON

### ISSUE THREAD

**Issue by mjeronimo** [nimble-code/Cobra#58]
I believe the intention is for the output file of Cobra to be a valid JSON file (although it currently has a .txt extension). 

To demonstrate that Cobra sometimes does not produce valid JSON:

Create two files, t1.c and t2.c with the following contents:

```C
#include <stdio.h>

int main()
{
  int x;
  int y;

  printf("%d:%d\n", x, y);
  return 0;
}
```
The run cobra on these files with:

```
$ cobra -C++ -comments -json -f p10 t1.c t2.c
```
The contents of the output file (\_P10\_.txt) will be:

```
[
  { "type"      :       "Rule 5: use minimally two assertions per function on average",
    "message"   :       "lines 3..10",
    "file"      :       "t2.c",
    "line"      :       3,
    "cobra"     :       "1 2 22"
  }
  { "type"      :       "Rule 5: use minimally two assertions per function on average",
    "message"   :       "lines 3..10",
    "file"      :       "t1.c",
    "line"      :       3,
    "cobra"     :       "1 2 22"
  }
]
```

Where there is not a comma between the entries.

**Comment by nimble-code** [comment]
confirmed -- this will be fixed in the next update to version 4.3
(this includes small updates also in the src_app/cwe tools)
the P10 output also showed some redundancies in that output -- will also be fixed in the 4.3 update
thanks for catching this!

**Comment by nimble-code** [comment]
new version Cobra 4.3 now uploaded, which should fix this and a couple of other issues
